make/kind_images.sh

```diff
@@ -20,6 +20,7 @@ KIND_IMAGE_K8S_122=docker.io/kindest/node@sha256:7d9708c4b0873f0fe2e171e2b1b7f45
 KIND_IMAGE_K8S_123=docker.io/kindest/node@sha256:ef453bb7c79f0e3caba88d2067d4196f427794086a7d0df8df4f019d5e336b61
 KIND_IMAGE_K8S_124=docker.io/kindest/node@sha256:577c630ce8e509131eab1aea12c022190978dd2f745aac5eb1fe65c0807eb315
 KIND_IMAGE_K8S_125=docker.io/kindest/node@sha256:cd248d1438192f7814fbca8fede13cfe5b9918746dfa12583976158a834fd5c5
+KIND_IMAGE_K8S_126=docker.io/kindest/node@sha256:5dd2cec6a164de6a0948d01bda4a8aa569d6320dfcd33a3f1433450dc3fcba31
```

Comment:
I see the reason for it, but equally those SHAs aren't really provided in a way which we can programmatically consume as far as I've seen.

I kinda figured that since the latest SHA worked when we tested it here, it's probably fine to use?

Reply:
I'd probably rather use the one they tell folks to use or have a comment here explaining why the SHAs we generate here are different and why their recommendation doesn't apply to us as else every next person looking at this will probably have the same question

Reply:
(but am also aware that we probably want this merged before EOD to avoid periodics failing over weekend)

Reply:
Ahhh I _really_ don't want to move backwards on automating this stuff :sob: It's hard enough to keep things up to date with the little automation we do have and it doesn't _seem_ like using the latest images breaks us, although I guess it could be related to some test flakes.

I think it would be helpful if Kind published the supported versions along with kind releases, maybe in a JSON file attached to the GitHub release.

I'll add 1.26 manually for now (and do the same in #5647) , but I guess we need to address this more generally down the line :sob:

Reply:
sorry to be nitpicky, agree about automation

> I think it would be helpful if Kind published the supported versions along with kind releases, maybe in a JSON file attached to the GitHub release

I'll see if I can raise this in #kind they're always very responsive

Reply:
Yeah I follow #kind a lot too, always impressed by it. Would it be worth us creating a cert-manager issue to track this and then linking to that issue in #kind? I'm not sure I'll have the bandwidth tonight though

Reply:
> sorry to be nitpicky, agree about automation

It's all good, I know you just want things to be better!

Reply:
> Would it be worth us creating a cert-manager issue to track this and then linking to that issue in #kind? I'm not sure I'll have the bandwidth tonight though

yes, that would ensure that we don't lose it
I'll try to create it tonight else Monday morning

Reply:
I've asked in #kind for now https://kubernetes.slack.com/archives/CEKK1KTN2/p1671446197465289

Reply:
x-post https://github.com/kubernetes-sigs/kind/issues/2965#issuecomment-1357951318